### ISSUE TITLE
Contact list issue -  Unknown Contact

### ISSUE THREAD

**Issue by Niklason** [jlippold/WifiSMS#25]
Hey i got a quiet enoying error so let me eqplain it..

It is that i live in sweden so all my numbers start whit +46730686655 (note fake number).
but in my contact list it is 0730686655 so when i open wifisms on the computer nearly all my contacts are unknown contacts.

**Comment by treason** [comment]
I think if you put 467 as your country code it'll link up correctly

**Comment by Niklason** [comment]
Yes it did work but is it any way to implement it to the program so the 07 automaticly is +47 or something like that so i dont need to waste loads of time editing my contact list?
